main.go

```diff
@@ -176,6 +178,9 @@ func registryCommand(ctx context.Context, args *RegistryCmd) (err error) {
 		registry.WithResolveTimeout(args.MirrorResolveTimeout),
 		registry.WithLocalAddress(args.LocalAddr),
 	}
+	if args.BlobSpeed != nil {
```

Comment:
We have this pattern in two places now. Maybe we should have a No Operation throttle as well and use the noop if BlobSpeed is nil. Then we don't have to care about this anywhere else.

Reply:
A noop throttler is just extra complexity and call stacks so lets not do that. I think it is better to keep it this way unless we make the argument a pointer instead to allow passing nil pointers.